splinepy/helpme/fit.py

```diff
@@ -52,30 +75,41 @@ def parameterize_line(fitting_points, n_fitting_points, centripetal):
         u_k[1:-1] = _np.cumsum(chord_lengths[:-1]).reshape(-1, 1)
         return u_k.reshape(-1, 1)
 
+    def parametrize_surface(reorganized_queries, centripetal):
+        sizes = reorganized_queries.shape[:2]
+        dim = reorganized_queries.shape[-1]
+        u_ks = []
+        for i in range(2):
+            tmp = []
+            for j in range(sizes[i]):
+                select = [slice(None)] * 2
+                select[i] = j
+                tmp.append(
+                    parameterize_line(
+                        reorganized_queries[tuple(select)].reshape(-1, dim),
+                        centripetal,
+                    )
+                )
+            u_ks.append(_np.mean(tmp, axis=0))
+
+        return u_ks
```

Comment:
This implementation is wrong, because the slices are set in the first loop to `[0,:]`, `[1,:]` etc...
This is also the reason the current tests fail.